string arm
---
bool success
float64[] pose